app/assets/stylesheets/print/_language_support.scss

```diff
@@ -0,0 +1,7 @@
+@media print {
+  // Targeting (right to left) rtl languages
+  main[lang="ar"] {   // Arabic
+    direction: rtl;
```

Comment:
Do we also need to include lang selectors for Hebrew and Urdu (similar to Whitehall)? https://github.com/alphagov/whitehall/pull/6448/files

```
main[lang="ar"], // Arabic
main[lang="he"], // Hebrew
main[lang="ur"] { // Urdu
```

Reply:
Forgot to push, updated.

Reply:
I was going to say that was quick! 😄